src/routes/(infoPages)/[topTierPage]/[...serviceGroupPage]/+page.svelte

```diff
@@ -143,6 +147,40 @@
 
 <div />
 
+{#if imageGallery?.images.length > 0}
+  {#if imageGalleryTitle}
+    <h2>{imageGalleryTitle}</h2>
+  {/if}
+
+  <div>
+    <!-- Display a maximum of 8 images (2 rows of 4) -->
+    <!-- TODO: Enhance with a "show more" button revealing lightbox gallery -->
+    {#each chunk( imageGallery.images.flatMap((i) => [i, i]), 4, ).slice(0, 2) as row}
```

Comment:
duplicating these for the purposes of seeing a full image gallery grid. will switch to this before merging
```suggestion
    {#each chunk( imageGallery.images, 4, ).slice(0, 2) as row}
```